packages/video_player/video_player_avfoundation/example/ios/RunnerTests/Info.plist

```diff
@@ -18,5 +18,10 @@
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+    <key>NSAllowsArbitraryLoads</key>
```

Comment:
is this required? if so, may have to inform developers about this change

Reply:
No it was for testing purposes of different free video files. I think I can remove them.